Assets/Scripts/Gamestate/PlayerManager.cs

```diff
@@ -253,14 +254,16 @@ public void SetPlayerInput(InputManager playerInput)
         {
             biddingPlayer.SetPlayerInput();
         }
-
-        ApplyColorFromIdentity();
     }
 
-    public void ApplyColorFromIdentity()
+    public void ApplyIdentity()
     {
         // Set player color
         meshBase.GetComponentInChildren<SkinnedMeshRenderer>().material.color = identity.color;
+        if (playerIK.TryGetComponent<BiddingPlayer>(out var biddingPlayer))
+        {
+            biddingPlayer.SetIdentity();
+        }
```

Comment:
Seems a bit weird to place this here, I suspect there might be a cleaner way to do this if we refactor playerfactory (probably a lot, so maybe ignore for this PR)

Reply:
Likely not that weird, as we would like to ensure that we listen to certain delegates from BiddingPlayer _only_ on the correct PlayerIdentity.